From e7de3f15bd1d8aaa47fb648ba4562b6c8f15e2cd Mon Sep 17 00:00:00 2001
From: endaaman <buhibuhidog@gmail.com>
Date: Fri, 13 Dec 2019 15:21:35 +0900
Subject: [PATCH 3/3] futures.InvalidStateError -> asyncio.InvalidStateError

---
 gbulb/transports.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gbulb/transports.py b/gbulb/transports.py
index c91c6e3..c99f750 100644
--- a/gbulb/transports.py
+++ b/gbulb/transports.py
@@ -163,7 +163,7 @@ class ReadTransport(BaseTransport, transports.ReadTransport):
         except asyncio.CancelledError:
             if not self._closing:
                 raise
-        except futures.InvalidStateError:
+        except asyncio.InvalidStateError:
             self._read_fut = fut
             self._cancelable.add(self._read_fut)
         else:
-- 
2.25.4

